### PR TITLE
inject request method

### PR DESCRIPTION
**Pull request by turbocrime**
temporary fix to mitigate missing request method

### CODE REVIEW

File: apps/extension/src/content-scripts/injected-penumbra-global.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ class PraxInjection {
   private stateEvents = new EventTarget();
 
   private injection: Readonly<PenumbraProvider> = Object.freeze({
+    request: () => this.postConnectRequest().then(() => Promise.resolve()),
```
Comment (grod220) [inline apps/extension/src/content-scripts/injected-penumbra-global.ts:60]: issue (blocking): can you add code documentation on the fact that this is temporary and should be removed after a pd point release?

**Review by TalDerei — APPROVED** [review]

File: apps/extension/src/content-scripts/injected-penumbra-global.ts (hunk 1 above)
Comment (turbocrime) [reply apps/extension/src/content-scripts/injected-penumbra-global.ts:66]: commented

**Comment by turbocrime** [comment]
i think this should be part of a fix release, rather than waiting on another solution.

File: apps/extension/src/content-scripts/injected-penumbra-global.ts (hunk 1 above)
Comment (turbocrime) [reply apps/extension/src/content-scripts/injected-penumbra-global.ts:66]: this was approved by another review, but you've left a blocking comment, so i will avoid merging this unless you re-review.

go ahead and merge this if you intend to include it in a release.

**Review by grod220 — APPROVED** [review]
Merging to include in next release

**Comment by grod220** [comment]
Will have to roll this one back. Looks like it works on first page load, but subsequent ones, it cannot make a connection: 

Screenshot 2024-08-26 at 11 50 49 AM

**Pull request merged**